src/test/resources/snapshots/com/conveyal/datatools/manager/jobs/DeployJobTest/canMakeOtp2GraphBuildAndServeManifestAndUserData-0.json

```diff
@@ -0,0 +1,25 @@
+{
+  "baseFolder" : "/var/otp/graphs",
+  "baseFolderDownloads" : [ {
+    "uri" : "s3://datatools-dev/test-deploy/build-config.json"
+  }, {
+    "uri" : "s3://datatools-dev/test-deploy/router-config.json"
+  } ],
+  "buildGraph" : true,
+  "graphObjUri" : "s3://datatools-dev/test-deploy/graph.obj",
+  "jarFile" : "/opt/otp/otp-latest-trimet-dev",
+  "jarUrl" : "https://opentripplanner-builds.s3.amazonaws.com/otp-latest-trimet-dev.jar",
```

Comment:
`jarUrl` should be `jarUri`.

Reply:
Fixed in https://github.com/ibi-group/datatools-server/pull/363/commits/d30a9edb8db1fe4acf1bc521ecae471938349f05